src/main/java/uk/gov/hmcts/opal/service/opal/NoteService.java

```diff
@@ -33,6 +35,7 @@ public class NoteService implements NoteServiceInterface {
 
     @Override
     @FeatureToggle(feature = "add-note", value = true)
+    @AuthorizedPermission(Permissions.ACCOUNT_ENQUIRY_NOTES)
```

Comment:
I'm not 100% on what you're doing with these annotations, but this looks wrong to me.
The check needs to be against a specific role, not any role within the user state.

Reply:
This is doing exactly the same check we had previously in our controllers code. 
https://github.com/hmcts/opal-fines-service/pull/317/files#diff-7fd3e88c8ec744c457c69d92339fe51bd7bd295b7b4f2ad9dbb85ca49e5c4ae0R30-R33 

Previously here

Reply:
This is the check for the 'addNote/saveNote' - https://github.com/hmcts/opal-fines-service/blob/22731cf62149435abce80604ccec4e3f5e26c3df/src/main/java/uk/gov/hmcts/opal/controllers/DefendantAccountController.java#L134-L136

Reply:
Oh yes, it's not needed here, this is different scenario though. removed.